cli/tests/cli-reference@.md.snap

```diff
@@ -773,6 +775,24 @@ If the given path is a directory, files in the directory will be visited recursi
 
 
 
+## `jj file track`
+
+Start tracking specified paths in the working copy
+
+Without arguments, all paths that are not ignored will be tracked.
+
+By default, Jujutsu starts tracking all new files automatically. This command is not useful then. You can configure which paths to automatically track by setting e.g. `snapshot.auto-track = 'none()'`. You will then need to run this command to start tracking new files.
```

Comment:
Does this say the same thing?
By default, Jujutsu tracks all files in the working copy automatically and this command does nothing (<--fix this?). If you manually configure the paths to automatically track by setting e.g. `snapshot.auto-track = 'none()'` then you will need to run this command to start tracking files.